pkgs/development/python-modules/python-language-server/default.nix

```diff
@@ -69,7 +69,9 @@ buildPythonPackage rec {
     "test_numpy_hover"
   ] ++ stdenv.lib.optional isPy27 "test_flake8_lint";
 
-  propagatedBuildInputs = [ setuptools jedi pluggy future python-jsonrpc-server flake8 ujson ]
+  buildInputs = [ flake8 ];
+
+  propagatedBuildInputs = [ setuptools jedi pluggy future python-jsonrpc-server ujson ]
     ++ stdenv.lib.optional (withProvider "autopep8") autopep8
```

Comment:
stdenv.lib is no longer recommended to use and should be replaced with lib. Could you change this?